test/domains/comet-classic/classic-contract.test.ts

```
@@ -247,6 +247,29 @@ interface StateObservation {
 }
 
+function routingMarkerLines(stdout: string): string[] {
+  return stdout.split('\n').filter((line) => /^(?:NEXT|SKILL|HINT):/.test(line.trim()));
+}
+
+function handoffMarkerLines(stderr: string): string[] {
+  return stderr.split('\n').filter((line) => /^\[(?:SET|HANDOFF)\]/.test(line.trim()));
+}
+
+function guardMachineLines(stderr: string): string[] {
+  const stripAnsi = (line: string) => line.replace(/\u001b\[[0-9;]*m/gu, '').trim();
```

ESLint

[error] Unexpected control character(s) in regular expression: \x1b.

(no-control-regex)